pagopa/io-app-design-system
# feat: [SIW-499] Add credentials catalog icons

## Short description
This PR adds three new icons for the IT-Wallet credentials catalog ([FIGMA](https://www.figma.com/file/c80PUA6BYANkGPyaqsRBNA/IT-Wallet---Esperienza-in-IO?type=design&node-id=2035-41538&mode=design&t=4RThxGUSOXVUc1Jo-0)): 
- disabilityCard;
- driverLicense;
- healthCard.

## List of changes proposed in this pull request
- Adds original `svg` files and React components;
- Adds icons mapping in `Icon.tsx`;

## How to test
Start the example app and open the `Icons` section, the new icons should be rendered properly.
